packages/frontend-2/pages/server-management/active-users.vue

```diff
@@ -0,0 +1,332 @@
+<template>
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink
+        :to="'/server-management'"
+        name="Server Management"
+      ></HeaderNavLink>
+      <HeaderNavLink
+        :to="'/server-management/active-users/'"
+        name="Active Users"
+      ></HeaderNavLink>
+    </Portal>
+
+    <h1 class="h4 font-bold mb-4">Active Users</h1>
+
+    <FormTextInput
+      size="lg"
+      name="search"
+      :custom-icon="MagnifyingGlassIcon"
+      color="foundation"
+      full-width
+      search
+      :show-clear="!!searchString"
+      placeholder="Search Users"
+      class="rounded-md border border-outline-3"
+      @update:model-value="debounceSearchUpdate"
+      @change="handleSearchChange"
+    />
+
+    <Table
+      :headers="[
+        { id: 'name', title: 'Name' },
+        { id: 'email', title: 'Email' },
+        { id: 'emailState', title: 'Email State' },
+        { id: 'company', title: 'Company' },
+        { id: 'role', title: 'Role' }
+      ]"
+      :items="users"
+      :buttons="[{ icon: TrashIcon, label: 'Delete', action: openUserDeleteDialog }]"
+      :column-classes="{
+        name: 'col-span-3',
+        email: 'col-span-3',
+        emailState: 'col-span-2',
+        company: 'col-span-2',
+        role: 'col-span-2'
+      }"
+      :overflow-cells="true"
+    >
+      <template #name="{ item }">
```

Comment:
Since `Table` is not supposed to be generic, you can properly type `item` (inside Table.vue) so that it's the correct type. Same goes for other templates here.